modules/web/src/app/shared/components/labels/style.scss

```
@@ -0,0 +1,26 @@
+// Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.hide-extra-labels {
+  height: 30px;
+  overflow: hidden;
+}
+
+.show-all-button {
+  width: 120px;
```

```suggestion
  min-width: 120px;
```
So that the text doesn't break on small screen resolution.